openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
That sounds reasonable, if we allow users to use scatter instead of nu-scatter there doesn't seem to be any other choice.  Though it does make sense to have `"nuP0"' be default since that is preferred.  

While we are at it, maybe we can improve on the `"P0"` and `"nuP0"` names.  Maybe `"out-scatter"` and `"nu-out-scatter"` or `"out-nu-scatter"`?  What do you think @wbinventor?